lib.py

```diff
@@ -48,34 +49,40 @@ async def benchmark(submit_msg: SubmitMessage):
                 result_lst = await run_code(op_name, op_id, tmpdir, in_file)
                 result = process_run_result(in_file, answers_map, result_lst)
                 results.append(result)
-            
+
             if len(results) > 0:
                 save_results(cur, op_id, day, part, code, results)
                 db.commit()
-            
+
         verified_results = [r for r in results if r.verified]
         if len(verified_results) > 0:
             median = stats.mean([r.median for r in verified_results])
             average = stats.mean([r.average for r in verified_results])
-            await msg.reply(
+            await ctx.reply(
                 embed=discord.Embed(
                     title="Benchmark complete (Verified)",
                     description=f"Median: **{ns(median)}**\nAverage: **{ns(average)}**",
-                    )
                 )
+            )
         else:
             median = stats.mean([r.median for r in results])
             average = stats.mean([r.average for r in results])
-            await msg.reply(
+            await ctx.reply(
                 embed=discord.Embed(
                     title="Benchmark complete (Unverified)",
                     description=f"Median: **{ns(median)}**\nAverage: **{ns(average)}**",
                 )
             )
 
-    except Exception:
-        logger.exception("Unhandled exception while benchmarking.")
-        await msg.reply(f"Unhandled exception while benchmarking day {day}, part {part}.")
+    except Exception as e:
+        logger.exception(f"Unhandled exception while benchmarking day {day}, part {part}.")
+        with io.BytesIO() as buf:
+            buf.write(bytes(''.join(
+                traceback.format_exception(e)), encoding='utf8'))
+            buf.seek(0)
+            errtxt = (f"Unhandled exception while benchmarking day {day}, part {part}: `{get_full_class_name(e)}`: "
+                      f"`{e}`")[:2000]
+            await ctx.reply(errtxt, file=discord.File(buf, filename="traceback.txt"))
```

Comment:
There's an open question whether we should do this or not. People can use `eprintln!` and then panic to leak the input file, which could be good or bad.

Reply:
ah
the existing ferris elf bot does give some traceback so i thought we would do the same, im not sure how that one filters for eprintln panics